src/js/port_handler.js

```diff
@@ -162,6 +162,7 @@ PortHandler.check_serial_devices = function () {
             currentPorts = self.updatePortSelect(currentPorts);
             self.selectPort(currentPorts);
             self.initialPorts = currentPorts;
+            self.portPickerElement.trigger('change');
```

Comment:
```suggestion
            GUI.updateManualPortVisibility();
```

Reply:
I replaced the other similar lines as well.

Reply:
![image](https://user-images.githubusercontent.com/8344830/192168581-bfd96e8e-459b-4833-b1bc-f4b36dfe0083.png)

On line 8, maybe revise the function to use toggle